jobs/iptables-manager/templates/bin/iptables-manager_ctl.erb

```diff
@@ -21,9 +21,12 @@ exec 2>> $LOG_DIR/$JOB_NAME.stderr.log
 
 case $1 in
   start)
-    iptables -F #Flush all existing rules
+    iptables -A INPUT -p ICMP --icmp-type 8 -s  "<%= p('allow_ips_list') %>" -j ACCEPT
```

Comment:
one small question here: what is icmp-type 8? I searched online and could understand it refers to 'echo' request of ICMP.
But here, what exactly is achieved by blocking this request?

Reply:
ICMP type 8 is ping.

Reply:
ok, thanks!